app/app.py

```diff
@@ -2,13 +2,14 @@
 
 config: Config = get_config()  # / production
 
-m: MicroService = MicroService('device')
+m: MicroService = MicroService("device")
 
 wrapper: DatabaseWrapper = m.get_wrapper()
 
-if __name__ == '__main__':
-    from resources.device import *
-    from resources.file import *
+if __name__ == "__main__":
+    import resources.device
+    import resources.file
+    import resources.hardware
```

Comment:
this won't load the endpoints properly. revert to `from ... import *` syntax

Reply:
but why It is only important that the interpreter runs over it and adds the endpoint that is the only purpose of these import statements. Further more it is better coding style.

Reply:
i just tested this again and it does not load the endpoints when using `import resources.device`. the microservice would return the unknown_service error when trying to call the endpoint. `from resources.device import *` seems to fix this. i also tried `from resources import device` but this does not work either. @SpartanerSpaten

Reply:
Ok I was net certain about it ... ok so revert the change